web/wp-content/plugins/tmbi-salesforce-dmp/inc/author-detail.php

```diff
@@ -0,0 +1,94 @@
+<?php
```

Comment:
Functions in this file have generic names without a prefix or namespace. Consider namespacing them, or prefix their names with the plugin name.